src/ledger/LedgerManagerImpl.cpp

```diff
@@ -1073,10 +1080,10 @@ LedgerManagerImpl::closeLedger(LedgerCloseData const& ledgerData,
     // Invoke completion handler on the _main_ thread: kick off publishing,
     // cleanup bucket files, notify herder to trigger next ledger
     auto completionHandler = [this, ledgerSeq, calledViaExternalize, ledgerData,
```

Comment:
Perhaps a more descriptive name/comment here would be nice, like `advanceLCLtoAppliedLedgerHandler` or something along those lines.

Reply:
done